src/components/modules/document/common/Upload/modules/Video/modules/Recorder/modules/RecorderControls/modules/SingleSlot/SingleSlot.view.vue

```diff
@@ -0,0 +1,10 @@
+<template>
```

Comment:
Which purpose for this component?

Reply:
for pass slot without any wrapper

Reply:
It's wierd for me as well. And we have div